net/metrics/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=metrics
+PKG_VERSION:=0.1.20170217
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache 2.0 License
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/metrics
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+lua
+  TITLE:=Prometheus-compatible Metrics Exporter
+  PKGARCH:=all
+  MAINTAINER:=Dan Luedtke <mail@danrl.com>
+endef
+
+define Package/metrics/description
+  A Prometheus-compatible metrics exporter for monitoring and statistics.
+endef
+
+define Package/acme/conffiles
```

Comment:
Done. Used ACME as template. Build still running. I'll let you know about the results.